### PR TITLE
Universally expunge nugatory `withSES` parameter

### PR DESCRIPTION
**Pull request by FUDCo**

### CODE REVIEW

**Review by Chris-Hibbert — APPROVED** [review]

**Review by warner — APPROVED** [review]
Remove the go.mod and we're all good. Thanks for taking on this task!

File: packages/cosmic-swingset/go.mod [hunk 1]
```diff
@@ -6,23 +6,16 @@ require (
 	github.com/Agoric/agoric-sdk v0.0.0-00010101000000-000000000000 // indirect
```
[the remark this answers is not in the document]
Comment (warner) [reply packages/cosmic-swingset/go.mod]: done!

**Comment by warner** [comment]
Oh, not worth changing, but it looks like `packages/SwingSet/test/test-devices.js` got chmodded to something different (`+x`?) during this patch. Not sure why, doesn't hurt anything, but you might keep an eye on your tooling in case it's doing something weird.

**Comment by FUDCo** [comment]
> Oh, not worth changing, but it looks like `packages/SwingSet/test/test-devices.js` got chmodded to something different (`+x`?) during this patch. Not sure why, doesn't hurt anything, but you might keep an eye on your tooling in case it's doing something weird.

Looks like a `g+x` got added.  That's troubling, because nothing should have touched that.  My first suspect is a typo while editing in emacs, but who knows what keystroke...

**Pull request merged**